README.md

```diff
@@ -210,7 +283,8 @@ You can also use Btune from C. Similar to the Python examples above, you can act
     // btune
     btune_config btune_config = BTUNE_CONFIG_DEFAULTS;
     //btune_config.perf_mode = BTUNE_PERF_DECOMP;
-    btune_config.tradeoff = .5;
+    btune_config.tradeoff[0] = .5;
+    btune_config.tradeoff_nelems = 1;
     btune_config.behaviour.nhards_before_stop = 10;
     btune_config.behaviour.repeat_mode = BTUNE_REPEAT_ALL;
     btune_config.use_inference = 2;
```

Comment:
Can you add an example of using a triplet in the tradeoff too?

Reply:
Also, now that `Blocksize` and `Shufflesize` are gone, it would be nice if you could update the tables that are output by `BTUNE_TRACE` and remove these fields.

Reply:
Fixed in 9fcfb27cc73198307be5bb26ef401e393db473a5.